eng/liveBuilds.targets

```diff
@@ -109,8 +109,6 @@
         <IsNative>true</IsNative>
       </RuntimeFiles>
 
-      <MonoCrossFiles Condition="'$(TargetsMobile)' == 'true'"
```

Comment:
Should reference to this item be removed from `src/libraries/externals.csproj` too?